tests/Tests/500__jupyterhub/custom-image.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Documentation    Testing the backend for custom images (Adding ImageStream to redhat-ods-applications)
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/GPU.resource
+Library          JupyterLibrary
+Library          OpenShiftCLI
+Suite Teardown   Custom Image Teardown
+
+
+*** Variables ***
+${YAML} =         configs/resources/custom_image.yaml
+${IMG_NAME} =     custom-test-image
+
+
+*** Test Cases ***
+Verify Custom Image Can Be Added
+    [Documentation]    Applies the YAML and Gets the ImageStream
+    ...                Then loads the spawner and tries using the custom img
+    [Tags]    Tier2
+    ...       ODS-1208
+    ${apply_result} =    Run Keyword And Return Status    Run    oc apply -f ${YAML}
```

Comment:
I would create a keyword `Import Custom Image And Check Status` to wrap lines 24-25

Reply:
Done in latest commit